packages/code-studio/src/settings/FormattingSectionContent.jsx

```diff
@@ -621,8 +675,8 @@ export class FormattingSectionContent extends PureComponent {
           const selectedFormat = DecimalColumnFormatter.makeFormat(
             '',
             e.target.value,
-            null,
-            DecimalColumnFormatter.TYPE_GLOBAL
+            DecimalColumnFormatter.TYPE_GLOBAL,
```

Comment:
Need to update the `makeFormat` call on line 650. Github won't let me comment on that line directly.

Reply:
Good catch